test/relayHub/RelayHub.test.ts

```diff
@@ -0,0 +1,1199 @@
+                    from: relayOwner
+                });
+
+                //Verifying stake is now unlocked
+                let stakeInfo = await relayHubInstance.getStakeInfo(
+                    relayManager
+                );
+                const isUnlocked = Number(stakeInfo.withdrawBlock) > 0;
+                assert.isTrue(isUnlocked, 'Stake is not unlocked');
+
+                //Moving blocks to be able to unstake
+                await evmMineMany(Number(stakeInfo.unstakeDelay));
+
+                stakeInfo = await relayHubInstance.getStakeInfo(relayManager);
+                const stakeBalanceBefore = toBN(stakeInfo.stake);
+
+                const relayOwnerBalanceBefore = toBN(
+                    await web3.eth.getBalance(relayOwner)
+                );
+
+                const gasPrice = toBN('60000000');
+                const txResponse = await relayHubInstance.withdrawStake(
+                    relayManager,
+                    { from: relayOwner, gasPrice }
+                );
+
+                //Getting the gas used in order to calculate the original balance of the account
+                const rbtcUsed = toBN(
+                    (await web3.eth.getTransactionReceipt(txResponse.tx))
+                        .cumulativeGasUsed
+                ).mul(gasPrice);
+
+                const relayOwnerBalanceAfter = toBN(
+                    await web3.eth.getBalance(relayOwner)
+                );
+                assert.isTrue(
+                    relayOwnerBalanceAfter.eq(
+                        relayOwnerBalanceBefore
+                            .sub(rbtcUsed)
+                            .add(stakeBalanceBefore)
+                    ),
+                    'Withdraw/unstake process have failed'
+                );
+
+                stakeInfo = await relayHubInstance.getStakeInfo(relayManager);
+                const stakeAfterWithdraw = toBN(stakeInfo.stake);
+
+                //Verifying there are no more stake balance for the manager
+                assert.isTrue(
+                    stakeAfterWithdraw.isZero(),
+                    'Stake must be zero'
+                );
+
+                relayWorkersBefore = await relayHubInstance.workerCount(
+                    relayManager
+                );
+                assert.equal(
+                    relayWorkersBefore.toNumber(),
+                    1,
+                    `Initial workers must be zero but was ${relayWorkersBefore.toNumber()}`
+                );
+
+                await assert.isRejected(
+                    relayHubInstance.relayCall(relayRequest, signature, {
+                        from: relayWorker,
+                        gas
+                    }),
+                    'RelayManager not staked',
+                    'Relay request was properly processed'
+                );
+            });
+
+            it('Should fail a relayResquest if the manager is in the last block of delay blocks', async () => {
```

Comment:
same misspelled here, we could update it in the whole file

Reply:
✅